kernel/src/fs/procfs/cmdline.rs

```diff
@@ -23,7 +23,10 @@ pub struct CmdLineFileOps;
 impl CmdLineFileOps {
     /// Create a new inode for `/proc/cmdline`.
     pub fn new_inode(parent: Weak<dyn Inode>) -> Arc<dyn Inode> {
-        ProcFileBuilder::new(Self).parent(parent).build().unwrap()
+        ProcFileBuilder::new(Self, InodeMode::from_bits_truncate(0o444))
```

Comment:
Some process files have such obvious permissions that you don't add a reference to them, right?

While I agree that they're somehow obvious, I think the code would be more maintainable if we added references to all of these files.